### PR TITLE
Initial app

### PR DESCRIPTION
**Pull request by lorrainepope**
This is the initial app. It has a sliding drawer menu.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,32 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "26.0.2"
+    defaultConfig {
+        applicationId "com.wherecycle.smartrecycle"
+        minSdkVersion 21
```
Comment (garygpope) [inline app/build.gradle:8]: Ok, so we're initially supporting Android 5.0+ only?

File: app/src/main/java/com/wherecycle/smartrecycle/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        FloatingActionButton fab = (FloatingActionButton) findViewById(R.id.fab);
+        fab.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Snackbar.make(view, "Replace with your own action", Snackbar.LENGTH_LONG)
```
Comment (garygpope) [inline app/src/main/java/com/wherecycle/smartrecycle/MainActivity.java:30]: This string should come from strings.xml as a resource

File: app/src/main/java/com/wherecycle/smartrecycle/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    }
+
+    @SuppressWarnings("StatementWithEmptyBody")
+    @Override
+    public boolean onNavigationItemSelected(MenuItem item) {
+        // Handle navigation view item clicks here.
+        int id = item.getItemId();
+
+        if (id == R.id.nav_camera) {
+            // Handle the camera action
```
Comment (garygpope) [inline app/src/main/java/com/wherecycle/smartrecycle/MainActivity.java:84]: Remove this comment, it's pretty pointless.

File: app/build.gradle [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
```
Comment (garygpope) [inline app/build.gradle]: compile has been deprecated, change to **implementation** 👍

**Review by garygpope — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (lorrainepope) [reply app/build.gradle:8]: Yeah, as you suggested I'd rather avoid multi dex issues.

File: app/build.gradle (hunk 4 above)
Comment (lorrainepope) [reply app/build.gradle]: Noted, I'll change that then

File: app/src/main/java/com/wherecycle/smartrecycle/MainActivity.java (hunk 2 above)
Comment (lorrainepope) [reply app/src/main/java/com/wherecycle/smartrecycle/MainActivity.java:30]: Yeah, I'll go back and change that

File: app/src/main/java/com/wherecycle/smartrecycle/MainActivity.java (hunk 3 above)
Comment (lorrainepope) [reply app/src/main/java/com/wherecycle/smartrecycle/MainActivity.java:84]: it's not doing any harm

Comment (garygpope) [reply app/src/main/java/com/wherecycle/smartrecycle/MainActivity.java:84]: It's not doing anything useful either :)

**Review by garygpope — COMMENTED** [review]
Good work, go ahead and squash-merge 👍

**Pull request merged**